odoo/addons/base/views/res_users_views.xml

```diff
@@ -270,6 +270,10 @@
                                     <field name="signature"/>
                                 </group>
                             </page>
+                            <!-- invisible container for calendar related info (to avoid bridge module just for xpath) -->
+                            <page string="Calendar" name="calendar" invisible="1" groups="base.group_system">
+                                <group name="calendar_accounts"/>
+                            </page>
```

Comment:
Code looks globally ok except this part.
If we want to add a placehoder container, let's put it in the "calendar" module instead of "base".

And maybe update the comment to:
`<!-- Placeholder container to hold information about external accounts (Google calendar, Microsoft calendar, ...) -->`

Reply:
Done